automation/src/main/java/org/wso2/testgrid/automation/parser/TestNgResultsParser.java

```diff
@@ -120,10 +121,16 @@ public TestNgResultsParser(TestScenario testScenario, String testLocation) {
      */
     @Override
     public void parseResults() throws ResultParserException {
-        final Path dataBucket = DataBucketsHelper.getOutputLocation(testScenario.getTestPlan());
+        Path dataBucket = DataBucketsHelper.getOutputLocation(testScenario.getTestPlan());
+        dataBucket = Paths.get(dataBucket.toString(), "test-outputs/scenarios", testScenario.getOutputDir(),
```

Comment:
"test-outputs/scenarios" can be replaced with TestGridConstants.TTEST_RESULTS_DIR